### DISCUSSION TITLE
Rose Gold 1.20.1 ??

### DISCUSSION THREAD

**Discussion by Foxgle22** [Luke100000/minecraft-comes-alive#970] [Q&A] [answered]
Rose Gold is in the version 1.20.1 or it's just a creative item?, if it is in the game, where I can find it? (sorry for my bad english)

**Comment by Luke100000** [comment, marked as answer]
Rose gold is an alloy: https://github.com/Luke100000/minecraft-comes-alive/blob/7356779484c9b0c296324a067095b5440fe8a33b/common/src/main/resources/data/mca/recipes/rose_gold_dust.json

**Reply by Foxgle22** [reply]
Thanks you!! :)
